.ci/spec/create-hosts.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+csource="${BASH_SOURCE[0]}"
+while [ -h "$csource" ] ; do csource="$(readlink "$csource")"; done
+root="$( cd -P "$( dirname "$csource" )/../" && pwd )"
+
+. "${root}/.ci/spec/env.sh"
+. "${root}/.ci/common.sh"
+
+pushd "${root}" > "${output}"
+
+# Assumes packet is already set up
+
+# job_id is provided by common.sh
+export PACKET_EXEC_REMOTE_DIRECTORY="${job_id}"
+
+# spec test configuration, defined by action runners, used by Vagrant on packet
+export PKT_VAGRANT_HOST_BOXES="${VAGRANT_HOST_BOXES}"
+export PKT_VAGRANT_GUEST_BOXES="${VAGRANT_GUEST_BOXES}"
+# other vagrant-spec options
+export PKT_VAGRANT_HOST_MEMORY=10000
```

Comment:
We should allow these to be set within the environment, and default to a reasonable value. This way we can easily adjust them using the env vars set in the workflow file.